islandora_bookmark.install

```diff
@@ -60,6 +60,12 @@ function islandora_bookmark_schema() {
         'length' => 255,
         'not null' => TRUE,
       ),
+      'descrip' => array(
+        'description' => 'The supplied description of the list',
+        'type' => 'varchar',
+        'length' => 255,
+        'default' => "Bookmark List",
```

Comment:
Should probably make this the empty string due to the retroactive nature of the updb below.